recipes/renet2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.1" %}
+
+package:
+  name: renet2
+  version: {{ version }}
+
+source:
+  url: https://github.com/sujunhao/RENET2/releases/download/v1.1-alpha/renet2-1.1.tar.gz
+  sha256: 106e15c10e7e050c1fd1aa6119c2f7551d39edc717e5e5f748392bce77c7f46b
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - renet2 = renet2.renet2:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - python=3.7
```

Comment:
This doesn't work on any other python version?

Reply:
no, it can work on python>=3.7.